YSE_App/models/frbtransient_models.py

```diff
@@ -227,14 +229,37 @@ def sum_top_two_PATH(self):
             path_values = path_values[argsrt]
             return np.sum(path_values[-2:])
 
+    @property
+    def mag_top_two_PATH(self):
+        """ PATH mag for the top two PATH P(O|x) values for the transient 
+
+        If there are more than 1, take the top two and 
```

Comment:
Do we really want the brightest? In `frb_status.py` it says: 
`            if frb.mag_top_two_PATH > mr_max:
                frb.status = TransientStatus.objects.get(name='TooFaint')
                frb.save()
                return`

But shouldn't this criterion be set by the faintest source? i.e., brightest source may be < mr_max, while faintest is not...

Reply:
seems right to me.  We are querying whether the sources are `too faint`.  

So, we find the brightest magnitude of the top two PATH and if even that is too faint, we call it too faint.

Am I missing something?